### DISCUSSION TITLE
Interest in a (paid) GUI version of osxphotos?

### DISCUSSION THREAD

**Discussion by RhetTbull** [RhetTbull/osxphotos#858] [Polls]
osxphotos has always been a hobby for me and my way of giving back to the open source community and that won't change.  However, I spend considerable time on it and GitHub stars don't pay for coffee.  I'm toying with the idea of building a GUI version of osxphotos that would be easier to use and not require the command line that would be a paid product.  Curios if there's interest in this.  My initial thought is a simple status bar app that you could configure to export one or more libraries and schedule to run automatically. Other ideas include a dynamic template builder with live preview for the directory and filename templates. 

Let me know if you would be interested in purchasing a GUI version of osxphotos and if there any specific features you'd like to see!

**Comment by yannikr** [comment]
Yes!  This would be an incredible app with an accompanying GUI.

I have been very impressed with PhotoSweeper X, Duplicate File Finder, and Powerphotos, but I find that they really lack the true power features that osxphotos provides.

**Reply by RhetTbull** [reply]
Which particular features of osxphotos would you be interested in? I don't want to duplicate what PowerPhotos and other tools can do as they've already got a loyal customer base (and more time than I have to work on their tools). Today, PowerPhotos can export though it lacks many of the options (and the templating system) that osxphotos. To me, an automated backup of your library that gives you full control over naming would be useful but interested in features others would like.

**Comment by ChZiegelt** [comment]
That sounds like something I've been thinking about since years ;-)
Unfortunately, my workflow is mostly on windows - coming with my own folder structure, exports from Google Picasa and currently migrating to Adobe Lightroom Classic or maybe Darktable or something alike.

My biggest problem has always been the iphone pictures from out of the handy / mac os photos - with keeping the meta data.

I'm currently working on a gui tool, written in delphi for multi platform to support my workflows.
I thought about integration your commandline tools for the macos part.

But I'm also trying to do most of the steps (renaming, relocating, updating metadata, .....) on the windows side.

I also play around with making use of Azure / AWS services for ai recognition of tags, keywoards, faces, ... for all those pictures which are not in my icloud / mac photos app.

Maybe there are some chances to cooperate ?

**Reply by RhetTbull** [reply]
Hi @ChZiegelt. I have something very specific in mind for the Mac app. Send me a note at rturnbull+git@gmail.com so we can compare ideas.
